translate/v3/create_glossary.go

```diff
@@ -29,7 +29,7 @@ import (
 func createGlossary(w io.Writer, projectID string, location string, glossaryID string, glossaryInputURI string) error {
 	// projectID := "my-project-id"
 	// location := "us-central1"
-	// glossaryID := "glossary-id"
+	// glossaryID := "your-glossary-display-name"
```

Comment:
As per the style in other Go files, `"my-glossary-display-name"`, see `projectID`.